Added an integration test for the Pytorch implementation of the DistilBERT model from issue #9948

danielpatrickhug opened the pull request:
# Adds Integration testing  for pytorch implementation of DistilBert from issue #9948

I implemented the test as described in the issue linked. I ran the test and it passed. I can extend the tests after confirmation of this current PR. Please let me know what you think. Thank you


Fixes #9948 


## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

LysandreJik commented:
Hello! Why did you close your branch? The integration test looks good, you only need to run `make fixup` at the root of your clone to apply the quality requirements.

danielpatrickhug commented:
Hi, sorry I as getting an error when i ran 'make fixup' and I was trying to figure it out. Ill finish it up tonight, unless you know whats wrong? Thank you for responding.

```
  File "utils/get_modified_files.py", line 28
    modified_files = subprocess.check_output(f"git diff --name-only {fork_point_sha}".split()).decode("utf-8").split()
                                                                                    ^
SyntaxError: invalid syntax
No library .py files were modified
  File "setup.py", line 192
    entries = "\n".join([f'    "{k}": "{v}",' for k, v in deps.items()])
                                            ^
SyntaxError: invalid syntax
```
